_emails/newsletter-2021-03.html

```diff
@@ -0,0 +1,148 @@
+---
+title: Origami Newsletter, March 2021
+companion_post_url: https://origami.ft.com/blog/2021/04/12/newsletter/
+---
+
+<!-- TL;DR -->
+{% capture text %}
+Updated Origami workshops. Work has begun to migrate Origami to npm. Reminder, we deprecate FT's Bower Registry in July. 2021 and plan to decomission in July 2022.
+{% endcapture %}
+
+{% include email/tldr.html content=text %}
+
+<!-- Top things -->
+{% include email/h2.html content="Top Things" %}
+
+{% capture text %}Some of the bigger Origami news from the last month:{% endcapture %}
+{% include email/markdown.html content=text %}
+
+
+
+<!-- Thing 1 -->
+{% include email/h3.html content="Update on the migration to npm (dropping Bower support)" %}
+{% capture text %}
+During January we announced our plan to [decommission the FT Bower Registry and migrate to the public npm registry](https://origami.ft.com/blog/2021/01/18/deprecating-bower-and-origami-via-npm/#who-does-this-affect).
+
+Work has now started on migrating the Origami components onto npm, 8 out of 66 of our component are now ready to be migrated. We aim to have all components, including documentation/guidance for how to migrate your products, complete by the start of Q3.
+
+As a reminder, below is a copy of our planned timeline to decommission the FT Bower Registry.
+
+1st July 2021 - Origami Components moved from Bower to npm and guides to migrate are written
+
+1st July 2021 - Deprecate the FT Bower Registry
+
+1st July 2022 - Decommission the FT Bower Registry
+
+{% endcapture %} {% include email/markdown.html content=text %}
+
+
+<!-- Thing 2 -->
+{% include email/h3.html content="New “Introduction to Origami components”workshop dates announced" %}
+{% capture text %}
+This workshop is great for people who are new to Origami or want a refresher on how to build with Origami.
+
+The workshop contents has had a revamp, it has been rewritten to be better suited for a remote audience and includes xyz and abc
+
+The dates we plan to run the workshop on are:
+- 42nd of borktober (include link to the Google Calendar entry)
+- 57th of Coffeeuary (include link to the Google Calendar entry)
+{% endcapture %} {% include email/markdown.html content=text %}
+
+
+<!-- Thing 3 -->
+{% include email/h3.html content="Guidance on how to customise an Origami component" %}
+{% capture text %}
+We now have a page to help our users decide if they need to customise an Origami component, and if so, how to go about it in a safe, supported way which should reduce the chance of the customisation breaking in the future.
```

Comment:
```suggestion
We now have a page to help Origami users decide if they need to customise an Origami component, and if so, how to go about it in a safe, supported way to avoid the customisation breaking in the future.
```

suggestion:
- swaps "our users" for "Origami users", think that sounds less like we own Origami i.e. collective project
- makes the customisation breakage bit more sure of itself, it definitely will prevent issues 😄